packages/web-api-scan-runner/src/setup-web-api-scan-runner-container.ts

```diff
@@ -15,5 +20,14 @@ export function setupWebApiScanRequestSenderContainer(): inversify.Container {
     registerScannerToContainer(container);
     registerServiceLibraryToContainer(container);
 
+    container.bind(iocTypeNames.ConvertAxeToSarifFunc).toConstantValue(convertAxeToSarif);
```

Comment:
This can be part of the scanner package like other register*() belongs to corresponding packages